site/docs/v1/tech/apis/tenants.adoc

```diff
@@ -21,6 +21,7 @@ The following APIs are provided to manage Tenants.
 
 * <<Create a Tenant>>
 * <<Retrieve a Tenant>>
+* <<Search for Tenants>>
```

Comment:
Should we standardize (at least on these new APIs with search) on the order?

https://github.com/FusionAuth/fusionauth-site/pull/1911/files#diff-4d0544aa40a149bba5144812d2f2bd30e279c675c0c0bf5292544b1e6322a7fcR22 is different (has search at the end).

I like CRSUD as the order, but don't really care as long as it is consistent.

Can @mmanes and @spwitt get together and make sure they are consistent?

Reply:
I reviewed my other PRs and made sure they were consistent.